File: ruby_related/naive_stack.rb
### First In First Out
### Write the class
class Stack

  def initialize
    # start new list, i.e. the array
    @stack = []
  end

  def push(val)
    # push val to back
    #   of list
    @stack.push(val)
  end

  def pop
    # return a val at the
    #   back of the list
    @stack.pop
  end

end

File: javascripts/naive_stack.js
// Write the constructor
function Stack() {
  // a list attribute
  this.stack = [];
} 

Stack.prototype.push = function(val){
  // push a val into the back of the list
  this.stack.push(val);
  // return this
  return this;
};

Stack.prototype.pop = function(){
  // remove a val from the back of the list
  this.stack.pop();

};